src/app/clusters/on-off/on-off.c

```diff
@@ -0,0 +1,276 @@
+/**
```

Comment:
This is duplicating code we have in the on-off-server directory already, I think.

Reply:
yes, was blindly-scripted cherry-pick based on the necessary callbacks.  I think we want on-off client, though, so will probably move your server work over.  make sense?

Reply:
whoops, no client :sheep:-ish

Reply:
removed